app/views/builds/index.html.erb

```diff
@@ -5,8 +5,21 @@
 <section class="tabs">
   <div class="row">
     <%= search_form do %>
-      <%= search_select :external, [["Yes", "true"], ["No", "false"]], size: 1 %>
```

Comment:
external was used to figure out how many of our builds were still built locally ... we might need that again if someone tries to find out if we still need the docker build host 🤷‍♂